src/navigation/FooterNav.scss

```diff
@@ -36,7 +36,6 @@
     width: auto;
     flex-shrink: 0;
     @screen lg {
-      @apply w-full;
```

Comment:
I think this will break Detroit, would you be able to make this a CSS variable? (Just this one style)